dashboard/app/models/lesson.rb

```diff
@@ -229,6 +229,11 @@ def summarize(include_bonus_levels = false, for_edit: false)
     lesson_summary = Rails.cache.fetch("#{cache_key}/lesson_summary/#{I18n.locale}/#{include_bonus_levels}") do
       cached_levels = include_bonus_levels ? cached_script_levels : cached_script_levels.reject(&:bonus)
 
+      description_student = I18n.t("data.script.name.#{script.name}.lessons.#{key}.description_student", default: '')
+      description_student = render_codespan_only_markdown(description_student) unless script.is_migrated?
```

Comment:
how are the scripts which _are_ migrated rendering this markdown?

Reply:
Still using `SafeMarkdown` but I believe the markdown is imported from CB at some point. Not completely sure but I didn't want to change the behavior for old scripts.

Reply:
Sorry I got your question backwards! The `SafeMarkdown` is already being used so it just renders as any other markdown we have.